### PR TITLE
getting-started/installing-node: add manual install instructions

### PR DESCRIPTION
**Pull request by williamboman**

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
given our release cycle, i think maintaining the specific tar will be too much. if you'd like to rewrite it so that it just point to the module and explains how to find the tar, i'd consider it. @williamboman

**Comment by williamboman** [comment]
I remember having a hard time finding the tarball, hence the PR :). Not sure how I eventually found it, it wasn't from official sources though. I suppose `$ npm view npm dist.tarball` would be the easiest way, but that assumes you have `npm` installed already.

Would just changing the URL to `https://registry.npmjs.org/npm/-/npm-{VERSION}.tgz` do it?

Also, out of curiosity, are there any plans to use a static site generator for generating the docs?

**Comment by ashleygwilliams** [comment]
hey @williamboman sorry for the delay in response!

i think that update to the URL is :+1: , if you make that change and rebase that'd be rad

re: static site generator... yes! i have a repo under my personal account where i am working on that :)
was thinking of using [metalsmith](http://www.metalsmith.io/). any thoughts?

**Comment by williamboman** [comment]
Rebased and updated.

> re: static site generator... yes! i have a repo under my personal account where i am working on that :)
> was thinking of using metalsmith. any thoughts?

I've only used Jekyll before so I wouldn't really know tbh.

File: content/getting-started/installing-node.md [hunk 1]
```diff
@@ -20,3 +20,9 @@ Node comes with npm installed so you should have a version of npm. However, npm
 `sudo npm install npm -g`
 
 Test: Run `npm -v`. The version should be higher than 2.1.8.
+
+## Installing npm manually
+
+> For more advanced users.
+
+The npm module is available for download at https://registry.npmjs.org/npm/-/npm-{VERSION}.tgz.
```
Comment (ashleygwilliams) [inline content/getting-started/installing-node.md]: final nit: could you put the address as an inline code block a la `https://registry.npmjs.org/npm/-/npm-{VERSION}.tgz`

**Comment by ashleygwilliams** [comment]
ok one final nit and then i''ll merge! thanks so much for contributing! would you like some npm socks? contributing to docs entitles you to some :smile:

File: content/getting-started/installing-node.md (hunk 1 above)
Comment (williamboman) [inline content/getting-started/installing-node.md]: oh, though I already had done that. updated

**Comment by williamboman** [comment]
> would you like some npm socks? contributing to docs entitles you to some

Haha sure. Does this apply to non-US people too (shipping etc.)?

**Comment by ashleygwilliams** [comment]
it does! :)

**Comment by ashleygwilliams** [comment]
should be on prod in a minute!

**Pull request merged**